### DISCUSSION TITLE
pdf report when run from command line

### DISCUSSION THREAD

**Discussion by FlorianProdinger** [vdemichev/DiaNN#1095] [Q&A]
Hello everyone,

I wanted to ask if someone knows how to generate the "pdf report" of QC statistics if DIA-NN is run from the command line? I could not find which option / parameter can be used to also generate the pdf report in the official readme. If someone knows the option to set, please let me know.
Thank you very much!

Florian

**Comment by vdemichev** [comment]
Hi Florian,

At the end of the analysis, the GUI prints the command it uses to envoke dia-nn-plotter.exe to generate the PDF report, i.e. need to copy this command.

Best,
Vadim

**Reply by FlorianProdinger** [reply]
thank you very much!
